### ISSUE TITLE
Improve OceanProfile Design

### ISSUE THREAD

**Issue by dhowe** [dhowe/spectre#582]
The OP looks good now, but I still feel it is a bit overwhelming to user, all the text and data in the same color/style. Can we use colors to differentiate between the different ocean icons and traits? Perhaps something like:

![image](https://user-images.githubusercontent.com/737638/75969697-47f87b80-5f0a-11ea-8db7-8cf121b8eca0.png)

**Comment by cqx931** [comment]
Something like this?
![Screenshot 2020-03-09 at 11 09 57 AM](https://user-images.githubusercontent.com/3056154/76228007-86d24e00-61f6-11ea-8e0f-a166220d923f.png)

**Comment by dhowe** [comment]
Better I think, but still quite subtle. Hmmm

**Comment by cqx931** [comment]
The lines of the icons and the sliders are very thin... I could also try one for the background of each trait, which should make it more obvious and easy to follow.

Sally

**Comment by cqx931** [comment]
![image](https://user-images.githubusercontent.com/3056154/76333042-7211bb00-62c7-11ea-86ed-5f99bce7f00b.png)

**Comment by dhowe** [comment]
Oh, I like that a lot more, though the yellow is a bit hard to read... what do you think??

**Comment by cqx931** [comment]
![image](https://user-images.githubusercontent.com/3056154/76347979-de97b480-62dd-11ea-85d2-48bbda026cbe.png)
![Screenshot 2020-03-10 at 2 47 12 PM](https://user-images.githubusercontent.com/3056154/76348105-08e97200-62de-11ea-8b51-ff451c2f0dc4.png)

A tiny bit darker yellow can help with readability issues, but bolder font-weight would more helpful in general.

**Comment by dhowe** [comment]
Yes, the top one is nice. Perhaps also bolding the header on the left side.
Another option might be a circle around the face of the dominant trait color.
